backend/src/main/resources/schema.graphqls

```diff
@@ -9,6 +9,17 @@ enum ResultValue {
   UNDETERMINED
 }
 
+enum UserPermission {
```

Comment:
Note that this is required (and will be enforced at startup) to stay in sync with the java enum that it reflects.